cluster/gce/container-linux/configure-helper.sh

```diff
@@ -82,6 +84,22 @@ function get-calico-typha-cpu {
   echo "${typha_cpu}"
 }
 
+# Prepare node labels specifically for master or node.
+function prepare-node-labels {
```

Comment:
Can we leave container-linux as-is here?

Reply:
Haven't addressed this yet. Not sure why container-linux should be treated specially?